src/mysql.c

```diff
@@ -1014,6 +1091,8 @@ static int mysql_read (user_data_t *ud)
 
 		gauge_submit ("cache_size", "qcache",
 				qcache_queries_in_cache, db);
+ 		gauge_submit ("cache_queries", "qcache-free-memory",
```

Comment:
Let's look  into another example:

`Innodb_buffer_pool_bytes_dirty / Innodb_buffer_pool_bytes_data`

http://dev.mysql.com/doc/refman/5.7/en/server-status-variables.html#statvar_Innodb_buffer_pool_bytes_data 

By such logic, whem also should be 'memory' type and two another type instances.

But I see no sence in mixing all (three) of them in one heap.